### PR TITLE
Do final part of mobile skeleton

### PR DESCRIPTION
**Pull request by jorgegoco**
In this branch I worked in the "works" and "about me" sections of the mobile skeleton.

### CODE REVIEW

File: head_mobile_portfolio.html [hunk 1]
```diff
@@ -32,6 +32,124 @@ <h2 class="big header-colored">I’m a software developer</h2>
 		</ul>
 		<img class="chevron" src="./images/Chevron.svg" alt="">
 	</main>
+	<h1 class="recent-works">My Recent Works</h1>
+	<hr class="short-green-line">
+	<section class="grid-container">
```
Comment (codepantha) [inline head_mobile_portfolio.html:37]: - [ ] The 'My Recent Works' heading should not be a `<h1>` in this case. The `<hx>` tags are hierarchical and as such, you can only use the `<h1>` for the main title of your document. In this case, please use some other `<hx>` tag 🙏

File: head_mobile_portfolio.css [hunk 2]
```diff
@@ -75,3 +75,222 @@ main {
...
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px;
+  background: #36b37f;
+  border-radius: 4px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  margin-top: 9px;
+}
```
Comment (codepantha) [inline head_mobile_portfolio.css:156]: - [ ] Please implement the button hover and active states as detailed in the Figma design 😎

File: head_mobile_portfolio.html [hunk 3]
```diff
@@ -32,6 +32,124 @@ <h2 class="big header-colored">I’m a software developer</h2>
...
+			<ul class="skills">
+			  <li class="language">Ruby on rails</li>
+			  <li class="language">css</li>
+			  <li class="language">JavScript</li>
+			  <li class="language">html</li>
+			</ul>
+			<span class="work-button">See Project</span>
+		  </div>
+		</div>
+	</section>
```
Comment (codepantha) [inline head_mobile_portfolio.html:110]: - [ ] Please make the 'See Project' button should be a `<button>` or a styled link 🙏 
- [ ] Also do same for all your buttons

File: head_mobile_portfolio.html [hunk 4]
```diff
@@ -32,6 +32,124 @@ <h2 class="big header-colored">I’m a software developer</h2>
...
+	<section class="about-me">
+		<div class="about-me-headline">
+		  <h2 class="about-me-title">About me</h2>
+		  <p class="about-me-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+			don’t hesitate to contact me.</p>
+		  <span class="about-me-button">Get My Resume</span>	
+		</div>
```
Comment (codepantha) [inline head_mobile_portfolio.html:117]: - [ ] Please make the 'Get My Resume' button a `<button>` or a styled link 😎 
- [ ] Let's ensure to follow the button styling specified in the Figma design 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @jorgegoco 👋, 

## STATUS: REQUESTED CHANGES ♻️ 
You have done a great job so far 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🎯 ⭐ 
- Meaningful commit messages ✔️ 
- Linter checks ✔️ 
- Beautiful UI ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jorgegoco** [comment]
Thanks.

**Review by codepantha — APPROVED** [review]
Hi @jorgegoco 👋, 

## STATUS: APPROVED ✔️ 

You have done a great job implementing all the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

### Optional

- [ ] Please remove the border on all your buttons so that they look exactly like the Figma design 😎 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jorgegoco** [comment]
Thanks.

**Comment by datgalcodes** [comment]
You're welcome😎

**Comment by codepantha** [comment]
You're welcome😎

**Pull request merged**